File: Majo/Act-Instructor_Henry/lenguajes/dart/junio/17-06.24.dart
import "dart:io";

void main(){

  print("Ingrese un nombre: ");
  String? nombre = stdin.readLineSync();

  if (nombre != null && nombre.length >= 2) {

    String dosUltimasLetras = nombre.substring(nombre.length - 2);
    print("Las dos últimas letras del nombre son: $dosUltimasLetras");

  } else {

    print("El nombre debe tener al menos dos caracteres.");
    
  }

}

File: Majo/Act-Instructor_Henry/lenguajes/ruby/junio/17-06-24.rb
puts "Ingrese un nombre:"
nombre = gets.chomp

if !nombre.nil? && nombre.length >= 2
    dos_ultimas_letras = nombre[-2..-1]
    puts "Las dos últimas letras del nombre son: #{dos_ultimas_letras}"
  else
    puts "El nombre debe tener al menos dos caracteres."
  end
  